src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java

```diff
@@ -51,7 +51,7 @@
  * Default value is {@code false}.
  * </li>
  * <li>
- * Property {@code fileExtensions} - Specify file type extension of files to process.
+ * Property {@code fileExtensions} - Specify the file type extension of files to process.
```

Comment:
Is this something added in this PR?

Reply:
It is value from setter, so it has to be the same in javadoc, test failure forcing us to do this

Reply:
I hope we will remove properties from javadoc one day completely.

Reply:
> value from setter, so it has to be the same in javadoc

Which we had the `XdocsJavadoc` doing before this migration. So why is it only being caught now?

Reply:
I see now. This is coming from the abstract class. `XdocsJavadoc` only looks at one file since it follows our limitations.

The `the` is coming from the field and the setter in the abstract class.